src/test/java/stepDefinition/SearchPageStepDef.java

```diff
@@ -14,13 +14,8 @@
 
     @Then("^'(.*)' list of items is displayed$")
     public void searchResultsList(String searchItem) {
-        Assert.assertTrue(getElementByName("searchProductGrid").findElement(By.xpath("//h2[@class='product-title']//a[contains(text(),'" + searchItem + "')]")).isDisplayed());
+        assertTrue("Search product grid is displayed",
+                getElementByName("searchProductGrid").findElement(By.xpath("//h2[@class='product-title']//a[contains(text(),'" + searchItem + "')]")).isDisplayed());
         getPage(getPageObjectName(SearchPage.class));
     }
```

Comment:
why would you keep a step definition class with just one method?

Reply:
cause it is related only to SearchPage